src/test/scala/com/bazaarvoice/sswf/CancelTest.scala

```diff
@@ -23,7 +23,7 @@ class CancelTestWorkflowDef(rememberer: Rememberer) extends WorkflowDefinition[S
 }
 
 class Rememberer {
-  var toRemember: String = null
+  var toRemember: String = _
```

Comment:
hmm so here is what happened..originally the value was null on this line. Idea code inspection recommended to change null to _. and if I use ??? the suite is skipped because the implementation is missing. I am keeping _ for now to get rid of the warning.